deploy/sre-pruning/115-pruning-rename-MUO-CR.CronJob.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-muo-cr-rename-sa
+  namespace: openshift-managed-upgrade-operator
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: osd-muo-cr-rename-role
+  namespace: openshift-managed-upgrade-operator
+rules:
+  - apiGroups:
+      - upgrade.managed.openshift.io
+    resources:
+      - upgradeconfigs
+    verbs:
+      - get
+      - list
+      - delete
+      - create
+      - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: osd-muo-cr-rename-rb
+  namespace: openshift-managed-upgrade-operator
+roleRef:
+  kind: Role
+  name: osd-muo-cr-rename-role
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: osd-muo-cr-rename-sa
+  namespace: openshift-managed-upgrade-operator
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: osd-muo-cr-rename
+  namespace: openshift-managed-upgrade-operator
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: osd-muo-cr-rename-sa
+          containers:
+            - name: osd-muo-cr-rename
+              imagePullPolicy: Always
+              image: image-registry.openshift-image-registry.svc:5000/openshift/cli:latest
+              args:
+                - /bin/bash
+                - -c
```

Comment:
probably add `-euo pipefail`
I don't see any dangerous statements in the commands, but when chaining oc commands that include delete statements I'd feel more comfortable if the script aborts when one of them fails.

Reply:
Thanks, great suggestion - added in commit d901b0f